### ISSUE TITLE
iPad drawing apps: maybe suggesting a non-vector one might also help?

### ISSUE THREAD

**Issue by luizaogs** [arch-kiosk/arch-kiosk-office#2200]
Hearing from Cyprus, and Anna said that everyone does seem to like Kiosk so far (as you both also already heard). She did say that the hardest thing for people to get used to has been sketching on the iPad, though. I haven't really had to use TouchDraw in a season myself and I can see the value of having something like it to develop more formal dayplans. But I can also see it being awkward to just do quick sketches of contexts, ones an excavator would just jot down quickly in a notebook if recording on paper. No? It could be just my impression, but maybe part of the awkwardness of TouchDraw is that it's a vector drawing tool. Might it be worth suggesting to users options of freehand apps they might want to use to quickly sketch things, too? In addition to TouchDraw, not as a replacement. I know Procreate is supposed to be quite good, for instance -- haven't used it myself but have seen it used and it looks like one is drawing on paper. 

Just a thought!

**Comment by lbestock** [comment]
Updates appreciated!

So honestly the easiest way to do this for a really quick sketch is just to use the signature function in Filemaker itself. It's the only way I would ever make a sketch for a locus relation, and it can be done for an image container. I also use the ipad's own image edit tools for drawing on photos as often as I do bring them into Touchdraw. 

But you know the real answer here is that yeah, learning software takes time and it is worth that time learning in order to do good work. It might be more intuitive and easy to sketch in Procreate or the signature function, and that might be useful sometimes. But to turn to that as a reason to not do the work to learn how to use a vector program, which has major advantages in this context in particular for any more detailed or accurate or iterative drawing you do, well, I cannot support that as a good plan :) I know you say in addition to, not instead of, but your work ethic is unusual and I don't take you as typical.

**Comment by urapadmin** [comment]
Procreate?
That's the name?

**Comment by lbestock** [comment]
Totally. Same one Christelle brought up as an alternative to doing digital epigraphy with vectors, so I'm a little sensitive :)

**Comment by urapadmin** [comment]
That's pretty feature rich itself, though. Is the basic image editing not sufficient for quick sketches? (https://www.youtube.com/watch?v=_VoV42KpI94)

**Comment by luizaogs** [comment]
Never mind. I forgot that one can use the signature function to sketch in other image containers than just relations. Because yeah, I don’t mean as a replacement to TouchDraw, but if I had to use TouchDraw for every single rough sketch I would be unhappy (not because I am lazy and do not want to learn to use it properly - but because I don’t think every rough sketch needs that level of time spent on it).

**Comment by urapadmin** [comment]
I think before we judge this from afar we should also try to get a clearer picture of what exactly makes it hard. I can imagine that perhaps getting the stuff in and out of FileMaker might be an issue that gets confused here with using the iPad to sketch in general. So let's try to get more detailed information once Katie is back (Anna?)

(I reopen this :o) so that we don't forget to ask)

**Comment by lbestock** [comment]
I'll also note that using the ipad's own editing tools to work on a blank canvas I didn't cover in that video and it isn't quite as intuitive (but still very possible and a good middle ground between the signature feature and a vector program). Also, @luizaogs, you unwittingly hit 47 raw nerves with that one that you couldn't have known about and should not take the fall for :)

**Comment by luizaogs** [comment]
> I think before we judge this from afar we should also try to get a clearer picture of what exactly makes it hard. I can imagine that perhaps getting the stuff in and out of FileMaker might be an issue that gets confused here with using the iPad to sketch in general. So let's try to get more detailed information once Katie is back (Anna?)

Yes, this seems to be at least part of it. I asked Anna if she had any more updates and she said someone had said that they wished they could draw plans in FM itself.

**Comment by urapadmin** [comment]
there is a new preview version on PVD. Download it onto an ipad, turn on test mode, grab your pen and hit the little red button next to some narrative in the unit narratives. Play with the circle.

**Comment by luizaogs** [comment]
Yeah, I can move the circle around, enlarge, etc. with my apple pencil just fine.

**Comment by urapadmin** [comment]
okay. Creating a fully fledged (even a decent) drawing tool like that would be a new job I did not apply for. We will have to reduce its capabilities to the max. Like: Rectangles, Ovals, lines. More than the signature tool but way less than touch draw. Needs some thinking: What is actually most important to draw these kinds of sketch maps in the field?

I guess it is worth exploring this. So, first step for me would be to make a proof of concept: create a drawing (will be an svg), save it as a dayplan, open it again, manipulate it.

**Comment by luizaogs** [comment]
Maybe wait to talk to Katie once she is back? I don't know, it sounds like a lot of work and I am not sure it is time you should be spending, especially since Laurel thinks vector drawing programs are the way to go. (I guess my point is: I wouldn't want you to waste time on something that isn't useful because I brought it up myself. Perhaps asking her directly will give you a better idea of what is needed, if anything.)

**Comment by lbestock** [comment]
Oh cool. I would never have suggested drawing in FM because I would have thought it prohibitively much work. If it isn't, I think it is absolutely worth working on. I don't think we need to wait for feedback from Katie's team because this really isn't project specific and anyway, first time users are often the least perceptive about what really matters or would help - they are just getting used to using this at all after all.

I think the two of you might be talking about different things, though, and that needs clarification first and foremost. Lutz do you mean a vector drawing program in FM, for which you've given us a red circle preview? Luiza it sounds like you are assuming a raster program in FM. Right? Are they equally possible? I have thoughts. I don't on first impression think you are going to get away from needing TD as a vector program and I think the desire to have "intuitive" drawing in FM is more a raster thing.

Immediate thoughts either way: we need to be able to have a grid behind it to draw on, and one needs to be able to erase. Erasing is totally different in vector/raster and thinking about that is what alerted me to the possibility you didn't mean the same thing.

**Comment by luizaogs** [comment]
I wasn't referring to one or the other specifically, really. But I guess if vector drawing is part of the issue for the simpler sketches and plans I was talking about, then, yeah, raster in FM and vector in TD (or wherever else) would make more sense. Unless the simplified nature of the FM drawing tool Lutz would come up with, if it is vector, would help get away from some of the difficulties of vector drawing programs like TD. If that makes sense...

**Comment by urapadmin** [comment]
While extending FM is not the goal anymore here, for Image Annotation I should keep in mind that we might want a raster layer on top of either nothing or an image. I know there is a raster mode in Fabric.js, that seems to embed a raster object in the vector format, much as SVGs can embed bitmaps. Perhaps that way the Designer of Image Annotation ca serve both worlds.

**Comment by urapadmin** [comment]
so I make that a new ticket for image annotation: #3549
